INSTALL.md

```diff
@@ -1211,34 +1216,8 @@ Unix, corresponding on other platforms) and the OpenSSL binary
 ("openssl").  The libraries will be built in the top-level directory,
 and the binary will be in the "apps" subdirectory.
 
-Troubleshooting:
```

Comment:
Troubleshooting is now a separate section, see https://github.com/openssl/openssl/pull/10545/commits/0fa6df71b3ae0e87ef32a6dc8629f298407ed710#diff-7d442b7eb49f5fc377f51e74b291cfc1R1549.